Libplanet.Explorer/Queries/Query.cs

```diff
@@ -51,7 +52,8 @@ public Query(BlockChain<T> chain, IStore store)
 
             while (limit is null || limit > 0)
             {
-                if (!excludeEmptyTxs || block.Transactions.Any())
+                if ((miner is null || miner == block.Miner) &&
+                    (!excludeEmptyTxs || block.Transactions.Any()))
```

Comment:
IMO this is more meaningful way. 
```suggestion
                bool isMinerValid = miner is null || miner == block.Miner
                bool isTxValid = !excludeEmptyTxs || block.Transactions.Any()
                
                if (isMinerValid && isTxValid)
```